### PR TITLE
Add Cohesion tracking to artist Top Works content

### PR DESCRIPTION
**Pull request by ashleyjelks**
Adds tracking with Cohesion schema.

Screen Shot 2020-06-11 at 4 05 20 PM

https://artsyproduct.atlassian.net/browse/FX-1987

### CODE REVIEW

File: src/v2/Apps/Artist/Components/ArtistTopWorksRail/ArtistTopWorksRail.tsx [hunk 1]
```diff
@@ -83,11 +97,19 @@ export const ArtistTopWorksRail: React.FC<ArtistTopWorksRailProps> = ({
...
+                      contextModule: ContextModule.topWorksRail,
+                      contextPageOwnerType: OwnerType.artist,
+                      destinationPageOwnerType: OwnerType.artwork,
+                      destinationPageOwnerId: id,
+                      destinationPageOwnerSlug: slug,
+                      horizontalSlidePosition: index + 1,
+                      type: "thumbnail",
+                      contextPageOwnerSlug: artist.slug,
+                      contextPageOwnerId: artist.id,
+                    }) as any
```
Comment (eessex) [inline src/v2/Apps/Artist/Components/ArtistTopWorksRail/ArtistTopWorksRail.tsx:111]: Will add a follow up PR for removing these `any` indicators soon!

File: src/v2/Apps/Artist/Components/ArtistTopWorksRail/__tests__/ArtistTopWorksRail.jest.tsx [hunk 2]
```diff
@@ -29,15 +34,57 @@ describe("Artist Top Works Rail Component", () => {
     expect(component.find(Carousel)).toHaveLength(1)
     expect(component.find(FillwidthItem)).toHaveLength(2)
   })
+
+  it("tracks the analytics properties when an artwork is clicked on the Top Works rail", () => {
+    const component = mount(<ArtistTopWorksRail {...props} />)
+    const elem = component.find(FillwidthItem).first()
```
Comment (eessex) [inline src/v2/Apps/Artist/Components/ArtistTopWorksRail/__tests__/ArtistTopWorksRail.jest.tsx:40]: i think adding `as FillwidthItem` might kill this type error

Comment (ashleyjelks) [reply src/v2/Apps/Artist/Components/ArtistTopWorksRail/__tests__/ArtistTopWorksRail.jest.tsx]: Good catch! Just updated it.

**Comment by sweir27** [comment]
Cool!! Cohesion seems great!

**Pull request merged**